pandas/tests/test_algos.py

```diff
@@ -251,6 +251,25 @@ def test_object_factorize(self, writable):
         tm.assert_numpy_array_equal(codes, expected_codes)
         tm.assert_numpy_array_equal(uniques, expected_uniques)
 
+    @pytest.mark.parametrize(
```

Comment:
Since this is only one set of parameters you don't need to parametrize - just move this down to the function body for clarity

Reply:
Done